config/locations.yml

```diff
@@ -79,7 +79,7 @@ concordia:
   locationurl: "http://library.concordia.ab.ca"
 concordsem:   
   name: "Concordia Lutheran Seminary"
-  locationurl: "​http://www.concordiasem.ab.ca/research/library_information.php​"
+  locationurl: "http://www.concordiasem.ab.ca/research/library_information.php"
```

Comment:
It may look like nothing has changed, but there are two zero width space characters at the start and end of the url that were removed